src/main/kotlin/step2/LottoNumber.kt

```diff
@@ -0,0 +1,43 @@
+package step2
+
+import java.lang.IllegalArgumentException
+
+class LottoNumber private constructor(
+    private var numbers: List<Int> = emptyList(),
```

Comment:
```suggestion
    private val numbers: List<Int> = emptyList(),
```
List와 같은 컬렉션은 add / remove 등 내부에서 데이터를 제어할 수 있습니다.
var 대신 val을 활용해보면 어떨까요?

Reply:
로또 번호는 오름차순으로 노출이 되어야합니다. 🙏